scintillometry/fourier/pyfftw.py

```diff
@@ -0,0 +1,156 @@
+# Licensed under the GPLv3 - see LICENSE
+
+import operator
+from .base import FFTMakerBase, FFTBase
+
+
+__all__ = ['PyfftwFFTMaker']
+
+
+# Only instantiate the class if FFTW exists.
+try:
```

Comment:
The `try/except` is not needed here, since you already do it in `fourier/__init__.py` (and it is OK to fail when someone attempts to import this directly).